### DISCUSSION TITLE
only_vlans_in_use and SVI

### DISCUSSION THREAD

**Discussion by PieterL75** [aristanetworks/avd#4501] [Q&A]
I have the "only_vlans_in_use" enabled on my l3leafs.
It works fine for all vlans that are defined on the connceted_endpoints,

but I also have an SVI that only use for my flowtracker. That vlans is not attached to any endpoints, thus also not created on the config. also the VRF related to that SVI vlan is not created

Is there a way to force a vlan to be created, with that only_vlans_in_use enabled and not have the vlan on an enpoint ?

**Comment by ClausHolbechArista** [comment]
We did not consider something like this when we built `only_vlans_in_use`. Are you using the SVI for "inband management" or only flow tracker specifically?

**Reply by PieterL75** [reply]
The management is all outofband with the mgmt interface and a dedicated VRF.

For the flowtracker, I have dedicated VRF, and only the SVI is part of that VRF. The tracker uses that SVI as the sourceinterface for traffic to the collector.
The VLAN of the SVI is not on any uplinks.

As a workaround, I added it to an uplink, but it would be more correct not having to do that.

**Reply by PieterL75** [reply]
I opened #4521 for this. ty for your input
